crates/shared/src/sources/balancer_v2/pools/weighted.rs

```diff
@@ -64,10 +64,6 @@ mod tests {
     use ethcontract::{H160, H256};
     use ethcontract_mock::Mock;
 
-    fn bfp(amount: &str) -> Bfp {
-        amount.parse().unwrap()
-    }
```

Comment:
I see, the macro is better than this... right?

Reply:
I wouldn't say its better... Its kind of arbitrary here in my opinion. The reason I went with a macro here is that:
- Its primarily used for testing
- No `use ...::bfp` in the test modules

A function would have been fine as well.